### PR TITLE
Upgrade to Node 10

### LINKED ISSUE #427: Update Psammead to use Node 10
**Is your feature request related to a problem? Please describe.**
Currently we are using Node 8 on Jenkins CI & for local development (.nvmrc). We should update to use Node 10 LTS. 

**Describe the solution you'd like**
Alongside [Simorgh Infrastructure](https://github.com/bbc/simorgh-infrastructure/issues/133) and [Simorgh](https://github.com/bbc/simorgh/issues/941) being updated, we need Psammead to be updated.

- Jenkinsfile to use Node 10 LTS container
- `.nvmrc` should have the updated node version (used in docker container)
- `.npmrc` should have the updated npm version (latest)
- All `package-lock.json` files should be updated (at the top level as well as at the package level) and all relevant package version numbers and changelogs should be updated as well. 

**Describe alternatives you've considered**
We could keep Node 8. However this is sub-optimal, since we want to have our latest documentation

**Testing notes**
[Tester to complete]

Dev insight: Running `npm run install:packages` with the latest node 10 and latest npm

**Additional context**
Certain tests have been observed to fail under Node 10 in simorgh. 
We should ensure the test suites psammead pass under Node 10.

As concluded in refinement, we should implement the Node 10 upgrade as follows
1. psammead.
2. simorgh + simorgh-infrastructure.

Additionally, we can consider checking that psammead components in Node 10 work
with simorgh + simorgh-infrastructure in Node 8. Lerna offers the ability to define this. We can look up this provision and use it.

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/psammead/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by DemonusPC**
Resolves #427 

**Overall change:** 
Updated the node version to Node LTS 10 

**Code changes:**

- Changed the nvmrc file to use the updated version
- Changed the node docker image used in the Jenkins pipeline
- Performed an installation using the new Node version.

---

- [x] I have assigned myself to this PR and the corresponding issues
- ~[ ] Tests added for new features~
- [ ] Test engineer approval

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -8226,20 +8226,20 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "resolved": "",
+          "resolved": false,
```
Comment (sareh) [inline package-lock.json]: Running this branch locally, with `npm i`, the resolved values throughout remain unchanged as `"resolved": "",`

Comment (DemonusPC) [reply package-lock.json]: I can rerun the npm install and recreate the package-lock file after the #511 is merged in

Comment (DemonusPC) [reply package-lock.json]: I nuked the package.lock file and reran the installation. Should be fine now

**Review by sareh — APPROVED** [review]
👍 Looks good. 
Locally, when using node 10.15.3, I have no regressions on the package-lock.json file. ✨

**Review by chris-hinds — APPROVED** [review]

**Comment by PriyaKR** [comment]
Regression looks good.

**Pull request merged**